draft-ietf-taps-transport-security.md

```diff
@@ -844,6 +855,70 @@ and the per-message nonce in the clear. Everything else is encrypted.
 
 - An unreliable transport protocol such as UDP.
 
+## OpenVPN
+
+OpenVPN {{OpenVPN}} is a commonly used protocol designed to replace IPsec. A
+major goal of this protocol is to provide an easy-to-use VPN. OpenVPN
+encapsulates either IP packets or Ethernet frames within a secure tunnel and
+can run over UDP or TCP.
+
+### Protocol Description
+
+OpenVPN facilitates authentication using either a pre-shared static key or
+using X.509 certificates and TLS. In pre-shared key mode, OpenVPN derives
```

Comment:
Is it possible to use a raw public key for authentication with TLS?

Reply:
Surprisingly, no. I looked again and found no reference in either the manual, any of the other pages, or the code.
Apparently they support TLS only with certificates.

Reply:
Okay, thanks!